azure/lb.sh

```diff
@@ -120,6 +120,25 @@ fn azure_lb_addresspool_delete(name, group, lbname) {
 	)
 }
 
+fn azure_lb_addresspool_get_id(addrpoolname, resgroup, lbname) {
+	out, err <= (
+		az network lb address-pool show
+					--resource-group $resgroup
+					--lb-name $lbname
+					--name $addrpoolname
+					--output json
+					>[2=]
+	)
+
+	if $err != "0" {
```

Comment:
I get the why of the redirect, but it's weird for the user to end up with an empty variable without aparent reason. So I think at least we should print an information saying there is no address pool for with that name for that load balancer.